### PR TITLE
update azure pipeline scaler doc with pod identity auth

### PR DESCRIPTION
**Pull request by toniiiik**
Update documenttion for azure pipeline scaler using PodIdentity https://github.com/kedacore/keda/pull/4867
 
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Review by JorTurFer — COMMENTED** [review]
This feature will be included as part of v2.13. Cloud you move the changes to that file?

**Comment by toniiiik** [comment]
> This feature will be included as part of v2.13. Cloud you move the changes to that file?

done

**Review by JorTurFer — APPROVED** [review]
LGTM! PTAL @tomkerkhove

File: content/docs/2.13/scalers/azure-pipelines.md [hunk 1]
```diff
@@ -227,3 +231,35 @@ spec:
             value: example-keda-template # Matches Scaled Job Parent
           
 ```
+
+### Example for PodIdentity authentication
```
Comment (tomkerkhove) [inline content/docs/2.13/scalers/azure-pipelines.md:235]:
```suggestion
### Example for using pod identity authentication
```

**Pull request merged**